apps/drec-api/src/models/DeviceGroup.ts

```diff
@@ -1,8 +1,34 @@
-import { IDevice } from '.';
+import { IDevice } from './Device';
+import {
+  CapacityRange,
+  CommissioningDateRange,
+  Installation,
+  OffTaker,
+  Sector,
+  StandardCompliance,
+} from '../utils/enums';
 
 export interface IDeviceGroup {
   id: number;
   name: string;
   organizationId: number;
+  fuelCode: string;
+  countryCode: string;
+  standardCompliance: StandardCompliance;
+
+  deviceTypeCodes: string[];
+  offTakers: OffTaker[];
+  installationConfigurations: Installation[];
+  sectors: Sector[];
+  gridInterconnection: boolean; // True - all devices have gridInterconnection true, if one has false, then this value is false
+  aggregatedCapacity: number; // total capacity of all devices beloning to this group
+
+  capacityRange: CapacityRange;
+  commissioningDateRange: CommissioningDateRange[];
+
+  yieldValue?: number; // ideally all underlying devices should have the same, otherwise take average?
+
+  labels?: string;
```

Comment:
why plural? It is single string. Just wondering

Reply:
Well, it can be an array of strings, but not sure how they will be added in the UI so I left them as a string column (label1, label2, label3 etc.). I used the same for devices so that`s why it`s the same here.